New post: Uploading And Downloading Your World files (#47)
FalixNodes-Software/help-center: _posts/minecraft/general/2021-08-31-Uploading-downloading-your-world-files.md

---
layout: post
title:  "Uploading And Downloading Your World Files"
categories: Minecraft
tags: mcgeneral
permalink: /minecraft/general/uploading-and-downloading-your-world-files/
---

# Uploading Your World
This guide explains how to upload a world to your server!

> Make sure your server is turned off while following the steps below.

> > Make sure your world is compressed (in ".zip", ".rar" formats or others) before following the steps below.

Reviewer: ```suggestion
> Make sure your world is compressed/archived (ending in ".zip", ".rar" or others) before following the steps below.
```

1. Setup your server. You can create a server on the [client panel](https://client.falixnodes.net).
2. Go to the [game panel](https://panel.falixnodes.net).
3. Click on your server.
4. Click on "File Manager" at the top of the page.
5. Click on "Upload", it is a big blue button located at the top right of the page.
6. Select your world archive.
7. After it is done uploading, locate your world archive and click on the 3 dots to its right.

> 8. Click on "unarchive".

Reviewer: ```suggestion
8. Click on "Unarchive".
```

9. Your world should be uploaded!

> Note: You can also use [SFTP](https://help.falixnodes.net/falix/general/sftp/)!

# Downloading Your World
This guide explains how to download a world from your server!

1. Go to the [game panel](https://panel.falixnodes.net).
2. Click on your server.
3. Click on "File Manager" at the top of the page.
4. Locate your world folder, it is usually called `world`.

> 5. Check the boxes to the left of your world folders.

Reviewer: > Locate your world folder, it is usually called `world`.

Here it's saying one world folder, while
> Check the boxes to the left of your world folders.

here it's saying multiple world folders.

Please stick with one or the other.

> 5. Check the boxes to the left of your world folder.

Reviewer: There is only one box next to a file, from what I see. What "boxes" are you talking about here?

Additionally, since you changed it to one folder, you can directly right click and hit "Archive".

Author: The "archive" popup only appears once you select a file

Reviewer: Right click --> "Archive"?

Author: doesn't it do the same thing? plus its easier to show in images.

Reviewer: It's easier to simply right click the file.

Author: how do I show it in the screen shot?

Reviewer: You don't need to use a screenshot, but I'd just say something like "right click file and click 'archive'" then show a screenshot of the Archive button.

This is just my opinion, do whatever you feel is best.

Author: In my opinion, I think using the box is better, for 2 reasons:
1- you can select multiple worlds,  (if you have more than one)
2- its easier to explain and show

Reviewer: 1. In your example, you stated one world. in addition, I've stated above to either set the example to multiple worlds or stick with one.
2. How? What's the point of clicking the tiny box next to the file and looking at the bottom of the page just to hit "Archive". I don't see how this is easier than simply right clicking the file.

Author: some servers might have multiple worlds as stated in the "note", should I go with multiple worlds for this guide?

Reviewer: Entirely up to you, but really, there isn't a difference from using the boxes or just right clicking each file if you're downloading multiple worlds.

> 5. Check the box to the left of your world folder.
> 6. Click on "archive" at the bottom of the page.
> 7. A new file should be created, locate it and click on the 3 dots to its right.
> 8. Click on "download".
> 9. Your world folder should be downloaded!

Reviewer: ```suggestion
5. Click on the 3 dots to the right of the folder.
6. Click on "Archive".
7. A new file should be created, locate it and click on the 3 dots to its right.
8. Click on "Download".
9. Your world folder should be downloaded!
```

> 9. Your world(s) folder should be downloaded!

Reviewer: The article name and section titles all state downloading **a** world (downloading **1** world).

Either change **everything**, or stick to one, which I recommend.

Author: the issue is that if I use only one world, users with multiple worlds might get confused

Reviewer: They can do the exact same for each world, that isn't confusing.

Reviewer 2: > 
> 
> They can do the exact same for each world, that isn't confusing.

Exactly.
